src/widgets/NumericTextCtrl.cpp

```diff
@@ -452,12 +452,12 @@ bool NumericTextCtrl::Layout()
 
    // Draw the background bitmap - it contains black boxes where
    // all of the digits go and all of the other text
-
+   wxPen Pen;
    wxBrush Brush;
 
    mBackgroundBitmap = std::make_unique<wxBitmap>(mWidth + mButtonWidth, mHeight,24);
    memDC.SelectObject(*mBackgroundBitmap);
-
+   
```

Comment:
whitespace - I see Matthieu removes them every PR, so let's not add it to his workload :P